tests/SideBySide/AppConfig.cs

```diff
@@ -19,18 +19,17 @@ public static class AppConfig
 				["Data:SupportsJson"] = "false",
 			};
 
-		private static string CodeRootPath = GetCodeRootPath();
+		public static string CertsPath => Config.GetValue<string>("Data:CertificatesPath")
```

Comment:
Just call `Path.GetFullPath(Config.GetValue<string>("Data:CertificatesPath"))` and it'll take care of convention